src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -646,7 +646,7 @@ services:
     form.type.common.translatable:
         class: 'PrestaShopBundle\Form\Admin\Type\TranslatableType'
         arguments:
-            - "@=service('prestashop.adapter.legacy.context').getLanguages()"
+            - "@=service('prestashop.adapter.legacy.context').getLanguages(false)"
```

Comment:
how about creating a new public function with a more relevant name instead: no way I can figure out what this function does with false parameter passed

Reply:
Well, I kind of agree with you. But the interface needs to be carefully thought through. Otherwise, we just add yet another useless interface in `Core`. :/ So, do you have any suggestions?

Reply:
`getActiveLanguages()` ?
I dont think we need to find some complex or exotic name, just describe what it does

Reply:
Should it be part of some interface?

Reply:
I'm not sure :/ this is only a shortcut function to make things easier, I'm not sure it's suitable for an interface

Reply:
That's why I didn't want to add yet another method without meaningful interface. But if you give me a green light on this, I'll do so (probably in `PrestaShop\PrestaShop\Adapter\LegacyContext`).

Reply:
Let's do it in `LegacyContext` 👍

Reply:
Done.